src/js/angular-tooltips.js

```diff
@@ -99,11 +119,13 @@
         $scope.showTooltip = function showTooltip () {
 
           theTooltip.addClass('_720kb-tooltip-open');
+          theTooltip[0].style.transition = "opacity " + speed + "ms linear";
```

Comment:
Here, I didn't add browser prefix. I tried on Chrome and FF. It works. Not sure on IE. I'll test it tomorrow. If you do, let me know :)